fastapi/app/api/project.py

```diff
@@ -0,0 +1,34 @@
+import random
+from logging import getLogger
+
+from app.models import Project
+from app.schemas import CreateProjectSchema
+from asgiref.sync import sync_to_async
+from config.exceptions import NotFoundException
+
+from fastapi import Request
+
+logger = getLogger(__name__)
+
+
+# FIXME: 使用する頻度に応じて別ファイルに切り出し
+def generate_id(length: int = 7) -> str:
+    return "".join(
+        random.choice("123456789abcdefghijkmnopqrstuvwxyzABCDEFGHJKLMNPQRSTUVWXYZ")
```

Comment:
hard coding はコードの意図が伝わりにくくなる要因になりがちなので、意味ある命名を与えてあげると good

```suggestion
        base58_strings =  123456789abcdefghijkmnopqrstuvwxyzABCDEFGHJKLMNPQRSTUVWXYZ
        random.choice(base58_strings)
```